src/VastManager.js

```diff
@@ -347,6 +347,15 @@ var vastManager = function () {
 					}
 					_vastRendererObj.playAd(xml, _options, firstVideoPreroll, _mobilePrerollNeedClick, prerollNeedClickToPlay, eventCallback);
 				}
+				else if (_options.adRenderer === 'custom') {
```

Comment:
Again, shouldn't you be comparing to a constant here, not the string (`_rendererNames.CUSTOM`, not `'custom'`)?